src/extensions/step_handlers/wim_handler/src/wimg.cpp

```diff
@@ -12,11 +12,30 @@ class WString
 public:
     WString(const char* utf8str)
     {
+        size_t utf8strLen;
+        HRESULT hr = StringCchLengthA(utf8str, STRSAFE_MAX_CCH, &utf8strLen);
```

Comment:
likely need `#include <strsafe.h>` to fix the first error.

Can use scripts/install-deps.ps1 and scripts/build.ps1 on a win vm to build locally, or just submit on the pipeline.

latest run for this pr on windows build pipeline has following build errors:
```
D:\a\1\s\src\extensions\step_handlers\wim_handler\src\wimg.cpp(16,48): error C2065: 'STRSAFE_MAX_CCH': undeclared identifier [D:\a\1\s\out\Debug\src\extensions\step_handlers\wim_handler\microsoft_wim_1.vcxproj]
D:\a\1\s\src\extensions\step_handlers\wim_handler\src\wimg.cpp(16,22): error C3861: 'StringCchLengthA': identifier not found [D:\a\1\s\out\Debug\src\extensions\step_handlers\wim_handler\microsoft_wim_1.vcxproj]
D:\a\1\s\src\extensions\step_handlers\wim_handler\src\wimg.cpp(24,84): warning C4267: 'argument': conversion from 'size_t' to 'int', possible loss of data [D:\a\1\s\out\Debug\src\extensions\step_handlers\wim_handler\microsoft_wim_1.vcxproj]
D:\a\1\s\src\extensions\step_handlers\wim_handler\src\wimg.cpp(31,22): error C2065: 'convertResult': undeclared identifier [D:\a\1\s\out\Debug\src\extensions\step_handlers\wim_handler\microsoft_wim_1.vcxproj]
D:\a\1\s\src\extensions\step_handlers\wim_handler\src\wimg.cpp(33,9): error C2065: 'convertResult': undeclared identifier [D:\a\1\s\out\Debug\src\extensions\step_handlers\wim_handler\microsoft_wim_1.vcxproj]
D:\a\1\s\src\extensions\step_handlers\wim_handler\src\wimg.cpp(33,105): error C2065: 'convertResult': undeclared identifier [D:\a\1\s\out\Debug\src\extensions\step_handlers\wim_handler\microsoft_wim_1.vcxproj]
D:\a\1\s\src\extensions\step_handlers\wim_handler\src\wimg.cpp(35,13): error C2065: 'convertResult': undeclared identifier [D:\a\1\s\out\Debug\src\extensions\step_handlers\wim_handler\microsoft_wim_1.vcxproj]
```